tensorrt_llm/_torch/pyexecutor/connectors/kv_cache_connector.py

```
@@ -613,6 +726,55 @@ def layer_pre_hook(self, module, *args):
+        failed_set = set(block_id for rank_ids in all_failed
+                         for block_id in rank_ids)
+
+        if not failed_set:
+            return []
+
+        affected = []
+        for req in active_requests:
+            try:
+                block_ids = kv_cache_manager.get_cache_indices(req)
+            except Exception:
+                continue
```

Ruff (0.15.17)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)